### ISSUE TITLE
refCollections does not work with react-bootstrap Select

### ISSUE THREAD

**Issue by Blystad** [QubitProducts/react-test-tree#43]
I'm unable to get `refCollections` to work with `react-bootstrap`s `Input` node (with `type=select`).

I just get a empty collection. However, if I replace `Input` with `select`, then it works correctly.

**Comment by oliverwoodings** [comment]
Would you mind creating a minimal fiddle to demonstrate this please?

**Comment by Blystad** [comment]
Here: https://gist.github.com/Blystad/b99264d4724f4c070948

I also created a repository with everything:

https://github.com/Blystad/issue-RTT-43

**Comment by oliverwoodings** [comment]
Thanks for this, looking into it for you now!

**Comment by oliverwoodings** [comment]
Released as v0.3.1

Thanks for the perfect test setup!

**Comment by Blystad** [comment]
The test still fails for me, even with 0.3.1.

**Comment by Blystad** [comment]
@oliverwoodings Can you please re-open this issue? 0.3.1 didn't fix it.

**Comment by oliverwoodings** [comment]
Yep, sorry I've been busy and haven't had time to look into it yet. This might end up being more complicated to solve than i first thought.

**Comment by oliverwoodings** [comment]
@Blystad sorry for the long delay on this. I've managed to find time to pick through why this doesn't work. The tl;dr of it is that react-bootstrap wraps the `<option />`s in several layers of parent components. Currently the refCollection detection system in react-test-tree does not support this deep nesting. I've got an idea for how to deal with this, but it probably won't be included until v0.4.0 since it involves rewriting the entire refCollection system.
